payload/builder.go

```diff
@@ -378,6 +385,56 @@ func (p *Payload) UnsetRelevanceScore() *Payload {
 	return p
 }
 
+// StaleDate defines the value stale-date for the aps payload
+// The date when the system considers an update to the Live Activity to be out of date.
+// ref: https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification
+//
+// {"aps":{"stale-date":1650998941}}
+func (p *Payload) StaleDate(staledate int64) *Payload {
```

Comment:
I think you should use camelCase for multi-word names.
```suggestion
func (p *Payload) StaleDate(staleDate int64) *Payload {
```